### PR TITLE
Changed launch page

### PR DESCRIPTION
**Pull request by EMerchant90**
Launch.storyboard updated to design specs

### CODE REVIEW

File: WalkingChallenge.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -630,10 +630,10 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = WalkingChallenge/WalkingChallenge.entitlements;
-				DEVELOPMENT_TEAM = 6SQ8F32998;
+				DEVELOPMENT_TEAM = 45YATE627L;
 				INFOPLIST_FILE = WalkingChallenge/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = com.akf.WalkingChallenge;
+				PRODUCT_BUNDLE_IDENTIFIER = com.akf.WalkingChalleng;
```
Comment (compnerd) [inline WalkingChallenge.xcodeproj/project.pbxproj]: Why this change?

File: WalkingChallenge/Assets.xcassets/Aga Khan Foundation LOGO_raster.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "Aga Khan Foundation LOGO_raster.png",
```
Comment (compnerd) [inline WalkingChallenge/Assets.xcassets/Aga Khan Foundation LOGO_raster.imageset/Contents.json:5]: I don't think that we need the `_raster` suffix to indicate that the image has been rasterized.

File: WalkingChallenge/Assets.xcassets/Aga Khan Foundation LOGO_raster_65.imageset/Contents.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "Aga Khan Foundation LOGO_raster_65.png",
```
Comment (compnerd) [inline WalkingChallenge/Assets.xcassets/Aga Khan Foundation LOGO_raster_65.imageset/Contents.json:5]: Similar with the `_raster_` infix.  Whats the `65` suffix for?

File: WalkingChallenge/Info.plist [hunk 4]
```diff
@@ -40,10 +16,56 @@
...
+	<key>FacebookDisplayName</key>
+	<string>AKF Causes</string>
+	<key>LSApplicationCategoryType</key>
+	<string>public.app-category.lifestyle</string>
+	<key>LSApplicationQueriesSchemes</key>
+	<array>
+		<string>fbapi</string>
+		<string>fbauth2</string>
+		<string>fbshareextension</string>
+		<string>fb-messenger-api</string>
```
Comment (compnerd) [inline WalkingChallenge/Info.plist:41]: I think that we should be able to drop the `fb-messenger-api` scheme (don't bother as a part of this change, its beyond the scope of this change).

File: WalkingChallenge/Info.plist [hunk 5]
```diff
@@ -2,30 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>NSAppTransportSecurity</key>
```
Comment (compnerd) [inline WalkingChallenge/Info.plist:5]: *sigh* I really dislike the PropertyList/CFDictionary serialization being based on `CFHash`.  What was the change made to the plist?  Perhaps you can edit it by hand instead to avoid the reordering noise?

File: WalkingChallenge/Launch.storyboard [hunk 6]
```diff
@@ -18,36 +18,44 @@
                         <viewControllerLayoutGuide type="bottom" id="xb3-aO-Qok"/>
                     </layoutGuides>
                     <view key="view" contentMode="scaleToFill" id="Ze5-6b-2t3">
-                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
-                            <imageView userInteractionEnabled="NO" contentMode="scaleToFill" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="splash" translatesAutoresizingMaskIntoConstraints="NO" id="RhP-6k-bd8">
-                                <rect key="frame" x="0.0" y="20" width="375" height="647"/>
+                            <imageView userInteractionEnabled="NO" contentMode="scaleToFill" horizontalHuggingPriority="251" verticalHuggingPriority="252" verticalCompressionResistancePriority="749" image="Aga Khan Foundation LOGO_raster" translatesAutoresizingMaskIntoConstraints="NO" id="IU1-63-VQT">
```
Comment (compnerd) [inline WalkingChallenge/Launch.storyboard:24]: Why the changes to the hugging priorities and the compression priorities?

File: WalkingChallenge/Launch.storyboard [hunk 7]
```diff
@@ -18,36 +18,44 @@
...
+                                <rect key="frame" x="44" y="160" width="327" height="179"/>
```
Comment (compnerd) [inline WalkingChallenge/Launch.storyboard:25]: Why are the bounds of the rect being adjusted?

File: WalkingChallenge/Launch.storyboard [hunk 8]
```diff
@@ -18,36 +18,44 @@
...
+                                </constraints>
                             </imageView>
-                            <label opaque="NO" clipsSubviews="YES" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Copyright © 2017 AKDN. All rights reserved." textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" minimumFontSize="9" translatesAutoresizingMaskIntoConstraints="NO" id="obG-Y5-kRd">
-                                <rect key="frame" x="19" y="606" width="338" height="21"/>
-                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
-                                <color key="textColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
+                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Steps4Change" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="jfe-el-kVe">
```
Comment (compnerd) [inline WalkingChallenge/Launch.storyboard:30]: Seems that the label text has been removed?  If so, lets just remove the label entirely.

**Review by compnerd — CHANGES_REQUESTED** [review]
I think that the pbxproj changes can probably be entirely dropped.  I'm not sure what the Info.plist changes are.

File: WalkingChallenge.xcodeproj/project.pbxproj (hunk 1 above)
Comment (EMerchant90) [reply WalkingChallenge.xcodeproj/project.pbxproj]: App won't run on my device because of a provisional profile issue - this was my workaround. This should've been changed back before committing.

File: WalkingChallenge/Assets.xcassets/Aga Khan Foundation LOGO_raster.imageset/Contents.json (hunk 2 above)
Comment (EMerchant90) [reply WalkingChallenge/Assets.xcassets/Aga Khan Foundation LOGO_raster.imageset/Contents.json:5]: probably not, this was just the name of the file when downloaded from UX source.

File: WalkingChallenge/Assets.xcassets/Aga Khan Foundation LOGO_raster_65.imageset/Contents.json (hunk 3 above)
Comment (EMerchant90) [reply WalkingChallenge/Assets.xcassets/Aga Khan Foundation LOGO_raster_65.imageset/Contents.json:5]: I felt that the image was too big and i resized by 65%, turns out it didn't matter. The name was just an indicator to myself of the changes made.

File: WalkingChallenge/Info.plist (hunk 5 above)
Comment (EMerchant90) [reply WalkingChallenge/Info.plist:5]: I'm not exactly sure how this change even happened, you seem to have better insight on this than I. If you could explain what is going on here I would appreciate it.

File: WalkingChallenge/Launch.storyboard (hunk 6 above)
Comment (EMerchant90) [reply WalkingChallenge/Launch.storyboard:24]: These changes were made to help with constraints. If it's too much or not correct, an indicator in the right direction on what you recommend for this would be great.

File: WalkingChallenge/Launch.storyboard (hunk 7 above)
Comment (EMerchant90) [reply WalkingChallenge/Launch.storyboard:25]: More constraints adjustments. Adjustments were made to keep the logo consistent in all screens. I may have approached this incorrectly, any advise would be great.

File: WalkingChallenge/Info.plist (hunk 5 above)
Comment (compnerd) [reply WalkingChallenge/Info.plist:5]: When you change the property list through Xcode, it will de-serialise and re-serialise it.  However, that is non-deterministic, and the keys can get re-ordered since the dictionary is unordered.  This means that its hard to spot the change.

File: WalkingChallenge/Launch.storyboard (hunk 6 above)
Comment (compnerd) [reply WalkingChallenge/Launch.storyboard:24]: Well, what happens if you drop them and let the default priorities take over?
